tests/syscalls/test_eip2333.c

```diff
@@ -0,0 +1,319 @@
+    .path = "3271528109/4135414213/1509797200",
+    .master_key = "528ca2018585232a3212491b7e015a6b3d3d7a2b5144c2ea6ab761d8ac8a1cff",
+    .private_key = "6dbe188e3e5f0b8fd5b83314924066ae4f43993f89d8dbaa36ed6edf237eaabe",
+    .path_len = 3
+  },
+  {
+    .curve = CX_CURVE_BLS12_381_G1,
+    .path = "835454411/3976701487/1774895332/1691472163/1650639831/2590215707/4082910115/3320782262/1914822140",
+    .master_key = "528ca2018585232a3212491b7e015a6b3d3d7a2b5144c2ea6ab761d8ac8a1cff",
+    .private_key = "61609898a95a58e0e82f3079e7b9d544a7c3bfb1764de89a5b823d8c07b23fec",
+    .path_len = 9
+  },
+  {
+    .curve = CX_CURVE_BLS12_381_G1,
+    .path = "3402326874/1631771538/3926620064/1135208208",
+    .master_key = "528ca2018585232a3212491b7e015a6b3d3d7a2b5144c2ea6ab761d8ac8a1cff",
+    .private_key = "3edb490e91c5ffdc054859272ce44a848c6e63d9e9caf90bac1ae099576ab97c",
+    .path_len = 4
+  },
+  {
+    .curve = CX_CURVE_BLS12_381_G1,
+    .path = "1133795433/843579193/2474203142/1396595878/2149462513",
+    .master_key = "528ca2018585232a3212491b7e015a6b3d3d7a2b5144c2ea6ab761d8ac8a1cff",
+    .private_key = "1c508b7c2e9c0cec8ae6fd6814cae3f386af8135e07445925f9b2d8fadbdf08e",
+    .path_len = 5
+  },
+  {
+    .curve = CX_CURVE_BLS12_381_G1,
+    .path = "2555287527/3388587387/3107743649/1989740811/3273976259/1516162297/2126678017",
+    .master_key = "528ca2018585232a3212491b7e015a6b3d3d7a2b5144c2ea6ab761d8ac8a1cff",
+    .private_key = "6e7d4b99701b44b1c3690a01dd603641a48a4c657cc72391a408da994db67d29",
+    .path_len = 7
+  },
+  {
+    .curve = CX_CURVE_BLS12_381_G1,
+    .path = "1018688040/1665183640/2867624941/672451332/2828286738/1242107022/2220222715/2326415856/1545775870/4232323255",
+    .master_key = "528ca2018585232a3212491b7e015a6b3d3d7a2b5144c2ea6ab761d8ac8a1cff",
+    .private_key = "0097ebe70b6bf42fdb3246fc9d2a9ac710bcd877fd0c349314ef2805a1021482",
+    .path_len = 10
+  },
+  {
+    .curve = CX_CURVE_BLS12_381_G1,
+    .path = "1366738206/3493596613/1309807041/1297120969/1523821670/2845978711/63646400/1173594587",
+    .master_key = "528ca2018585232a3212491b7e015a6b3d3d7a2b5144c2ea6ab761d8ac8a1cff",
+    .private_key = "4762a085a100411d5d4675938726dfd944826a90e75d620fff01cfba065b9135",
+    .path_len = 8
+  },
+  {
+    .curve = CX_CURVE_BLS12_381_G1,
+    .path = "1717688574/1427918720/2117515491/3667484657/333362278",
+    .master_key = "528ca2018585232a3212491b7e015a6b3d3d7a2b5144c2ea6ab761d8ac8a1cff",
+    .private_key = "05043ba7731cd27775017fe409d9632f638c2923ba7a15091d4d2c310a6b0896",
+    .path_len = 5
+  },
+  {
+    .curve = CX_CURVE_BLS12_381_G1,
+    .path = "1429896867/3196812878",
+    .master_key = "528ca2018585232a3212491b7e015a6b3d3d7a2b5144c2ea6ab761d8ac8a1cff",
+    .private_key = "55354ecb9c4d9de10cc61765334771b0963bd48c9074f675b520c9d27289a111",
+    .path_len = 2
+  },
+};
+
+// Same function as 'get_bip32_path'
+static int get_path(const char *str_, unsigned int *path, int max_path_len)
```

Comment:
If `get_path` is identical to `get_bip32_path`, please create a function that can be used in both files (maybe in `tests/syscalls/utils.c`?).